openpype/hosts/houdini/plugins/publish/collect_arnold_rop.py

```diff
@@ -126,8 +126,12 @@ def replace(match):
             return path
 
         expected_files = []
-        start = instance.data["frameStart"]
-        end = instance.data["frameEnd"]
+        start = instance.data.get("frameStartHandle") or \
+            instance.data["frameStart"]
+
+        end = instance.data.get("frameEndHandle") or \
+            instance.data["frameEnd"]
```

Comment:
If we are now **always** collecting frame start handle + frame end handle then why not rely on `frameEndHandle` and `frameStartHandle`.

Also, this logic here will fail if `frameStartHandle` is `0`, it should be: `instance.data.get("frameStartHandle", instance.data["frameStart"])` but because we're now just always collecting that data I'd actually just turn this into `instance.data["frameStartHandle"]` to be explicit.

**Additionally** - can we have an attribute on the instances that would be a `BoolDef` that states:
"Add asset frame handles" because we usually don't want the asset level to define whether a particular instance publish includes the shot's handles - we're not using the handles to define that kind of behavior for Houdini instances so we'd like to default that to 0.

Meaning that there'd be a boolean logic in the collector:
```python
if include_handles:
    handle_start = asset_data["handleStart"]
    handle_end = asset_data["handleEnd"]
else:
    handle_start = 0
    handle_end = 0
```

Reply:
`BoolDef` is a good idea but I need to mention that 
There are many plugins with different `order` that collect frame range
- collect_instance_frame_data.py
- collect_rop_frame_range.py